ckanext/canada/templates/internal/scheming/package/snippets/resource_form.html

```diff
@@ -57,7 +57,7 @@
     {% endif %}
   </div>
 
-  <div class="mrgn-bttm-lg">
+  <div class="mrgn-bttm-lg hidden">
```

Comment:
instead of hiding the form do we prefer to have the same set of rules for all resources? We could do that by removing this from the form and having an `output_validator` in our schema that always returns something like `{}`

Or we could remove the field completely and make ckanext-validation not require a value from the user

Reply:
Yeah, this was the easiest/quickest way to do it. As our schema uses the preset from Validation. We could copy the schema from validation and just change the form output value?

Or are we able to specific preset AND a form output snippet as well?

Reply:
Okay, so we need to have the `validation_options` in the form for sure, as there is the whole thing with the default validation options to set the max row limit.

So I have now just made some presets in our `presets.yaml` and used the hidden snippet. I have also changed it so we are not overriding the `validation_options_validator` and `resource_schema_validator` methods. So the validation plugin loading priority will not matter now.

@wardi  lemme know if you think it would be better to just remove the fields fully, and we can just hard code the row limit into our validation fork??

Reply:
@wardi nevermind, these fields are annoying me greatly right now, so I am going to remove them and just force the row limit json into the `validation_options` field value.

Reply:
@wardi okay...not sure if that is the best way to "remove" a field or not. But as the validation extension is still expecting these fields, I have just made harsh validator methods which always forcefully set the field values to what we want.

I am very bad with these schema yaml files, so lemme know. Thanks!

Reply:
please do the __extras thing as a separate PR just in case it causes problems with something